diff --git a/node_modules/letterparser/lib/parser.js b/node_modules/letterparser/lib/parser.js
index 487ecd9..657f4ca 100644
--- a/node_modules/letterparser/lib/parser.js
+++ b/node_modules/letterparser/lib/parser.js
@@ -225,6 +225,9 @@ function parseBody(depth, lines, lineStartIdx, lineEndIdx, lookaheadBoundaryLine
                     case 'base64':
                         {
                             var decoder = new TextDecoder(parsedType.encoding);
+                            body = '';
+
+                            stringBody = stringBody.replace(/\r?\n|\r/, '')
                             body = decoder.decode(base64_js_1.toByteArray(stringBody));
                         }
                         break;
@@ -236,8 +239,10 @@ function parseBody(depth, lines, lineStartIdx, lineEndIdx, lookaheadBoundaryLine
             else {
                 switch (headers['Content-Transfer-Encoding'].toLowerCase()) {
                     case 'base64':
-                        body = base64_js_1.toByteArray(stringBody);
-                        break;
+                      stringBody = stringBody.replace(/\r?\n|\r/, '')
+                      body = base64_js_1.toByteArray(stringBody);
+
+                      break;
                     case 'quoted-printable':
                         body = lettercoder_1.decodeQuotedPrintable(stringBody, parsedType.encoding);
                         break;
