diff --git a/node_modules/vuex-persist/dist/esm/index.js b/node_modules/vuex-persist/dist/esm/index.js
index fca7ef5..1becd3c 100644
--- a/node_modules/vuex-persist/dist/esm/index.js
+++ b/node_modules/vuex-persist/dist/esm/index.js
@@ -142,7 +142,7 @@ class VuexPersistence {
         this.filter = options.filter || ((mutation) => true);
         this.strictMode = options.strictMode || false;
         this.RESTORE_MUTATION = function RESTORE_MUTATION(state, savedState) {
-            const mergedState = merge(state, savedState || {}, this.mergeOption);
+            var mergedState = merge(state, savedState || {}, this.mergeOption);
             for (const propertyName of Object.keys(mergedState)) {
                 this._vm.$set(state, propertyName, mergedState[propertyName]);
             }
@@ -244,7 +244,7 @@ class VuexPersistence {
              * @param {Store<S>} store
              */
             this.plugin = (store) => {
-                const savedState = this.restoreState(this.key, this.storage);
+                var savedState = this.restoreState(this.key, this.storage);
                 if (this.strictMode) {
                     store.commit('RESTORE_MUTATION', savedState);
                 }
